lib/elixir/lib/range.ex

```diff
@@ -21,6 +21,22 @@ defmodule Range do
       iex> last
       3
 
+  A Range includes members from Enum such as reduce, count and member?
```

Comment:
I assume you're referring to the function listed in `defimpl Enumerable, for: Range do`.
Actually Range can be given as argument to any of the Enum functions.
We probably could simple mention that Range provides an implementation for Enumerable protocol, not sure about this though since `Map` and `List` moduledoc doesn't mention that fact.